src/IO/WriteBufferFromS3TaskTracker.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "config.h"
+
+#if USE_AWS_S3
+
+#include "WriteBufferFromS3.h"
+
+namespace DB
+{
+
+class WriteBufferFromS3::TaskTracker
+{
+public:
+    using Callback = std::function<void()>;
+
+    explicit TaskTracker(ThreadPoolCallbackRunner<void> scheduler_);
+    ~TaskTracker();
+
+    static ThreadPoolCallbackRunner<void> syncRunner();
+
+    bool isAsync() const;
+    void getReady();
+    void getAll();
```

Comment:
Why doesn't the function `getAll()` return anything. It's called `get all` so probably it should return all of something. Or maybe it's just a comment needed here.